src/main/java/Duke/ProgramException.java

```diff
@@ -0,0 +1,11 @@
+package Duke;
+/**
+ * Class to catch exceptions found when using Duke.
+ * @author Bryan Juniano
+ */
+
+public class ProgramException extends Exception {
+    public ProgramException(String exception) {
```

Comment:
Having the parameter be of type `String` but have parameter name `exception` may be a bit misleading (one may expect the constructor to accept an object of type `Exception` instead). Maybe using `String message` would be better?